Source/santactl/Commands/SNTCommandRule.m

```diff
@@ -61,15 +62,21 @@ + (NSString *)longHelpText {
           @"                   Does not work with --check. Use the fileinfo verb to check.\n"
           @"                   the rule state of a file.\n"
           @"    --sha256 {sha256}: hash to add/remove/check\n"
+          @"    --type {ruletype}: type of rules to remove\n"
           @"\n"
           @"  Optionally:\n"
           @"    --certificate: add or check a certificate sha256 rule instead of binary\n"
-          @"    --message {message}: custom message\n");
+          @"    --message {message}: custom message\n"
+          @"    --force: allow manual changes even when SyncBaseUrl is set\n");
 }
 
 - (void)runWithArguments:(NSArray *)arguments {
   SNTConfigurator *config = [SNTConfigurator configurator];
-  if ([config syncBaseURL] && ![arguments containsObject:@"--check"]) {
+  // TODO: --force flag was added so that we could manually add compiler rules during testing.
```

Comment:
You can add the --force flag in an `#ifdef DEBUG` block so it can only be used on debug builds.

Reply:
Done.